pnet_transport/src/lib.rs

```diff
@@ -269,6 +281,19 @@ impl TransportSender {
         set_sockopt_u8(self.socket.clone(), level, name, tos as u8)
     }
 
+    /// Configures path MTU discovery on a socket.
+    ///
+    /// On Linux-based systems, configuring the socket's path MTU discovery setting is
+    /// the way to "(un)set" the "don't fragment" bit in the IP header.
+    #[cfg(target_os = "linux")]
+    pub fn configure_path_mtu_discovery(&mut self, enable_disable: bool) -> io::Result<()> {
```

Comment:
Is there a reason to name this differently to the other methods that set options?

Reply:
I thought about this a ton and thought of two reasons:
1. It's how the option is "labeled" in Linux
2. The knock-on effect of changing the IP header option is just that: a knock-on effect.

That said, I agree with your sentiment because I added the option *just* so that I could set that bit in the IP header.

I will ultimately defer to you as the expert!!